### ISSUE TITLE
Meetup attendance isnt reliable

### ISSUE THREAD

**Issue by ondrae** [OpenTwinCities/cfapi_attendance_sync#1]
Hey @wbushey I spent the last few days adding your work to the CfAPI. Unfortunately, I've been unable to get meaningful attendance data out of Meetup. The attendance attribute on Meetup is always "attended" unless some group administrator uses a special tool to mark that person as "noshow". I imagine that never happens. I haven't seen it.

[Meetup acknowledged that this wasn't working](https://groups.google.com/d/msg/meetup-api/lRhFYAQDJFg/2OrbGop58MYJ) or truthful, so they are attempting to change it. It doesn't actually work right now though.

For that reason I'm going to pull out Meetup code and data from the `/attendance` endpoint of the CfAPI. Turns out the rsvp data is pretty big and historical though. I'm going to [give it its own endpoint](https://github.com/codeforamerica/cfapi/issues/243) and fancy graphs

**Comment by wbushey** [comment]
Yeah, Meetup's attendance tracking does leave something to be desired, and it is good to see they are fixing issues. For Open Twin Cities, using the Attendance Editor has worked pretty well, but the experience of adding an attendee who isn't already a member of the Meetup group is a little problematic (either you capture no information, or you ask the person to join.)
